ivadomed/main.py

```diff
@@ -274,15 +285,9 @@ def run_command(context, n_gif=0, thr_increment=None, resume_training=False):
 
         # Update threshold in config file
         context["postprocessing"]["binarize_prediction"] = {"thr": thr}
+        save_config_file(context, log_directory)
 
     if command == 'train':
-        # Save config file within log_directory and log_directory/model_name
```

Comment:
I was just wondering whether there was a reason why we put this "save config" in the `command = train` condition, ie not saving it at testing time.... But can't think of anything so... all good!